app/controllers/api/v1/puppetclasses_controller.rb

```diff
@@ -16,8 +16,8 @@ def index
           authorized(:view_puppetclasses).
           search_for(*search_options).paginate(paginate_options).
           select([:name, :id]).
-          includes(:lookup_keys)
-        render :json => Puppetclass.classes2hash(values.all)
+          includes(:lookup_keys).to_a
```

Comment:
in AR 4.1 the #all method has been changed. to_a does it. this is part of
the deprecation warnings cleanup, I think...